botocore/utils.py

```diff
@@ -3315,3 +3325,77 @@ def _serialize_if_needed(self, value, iso=False):
                 return value.isoformat()
             return value.strftime('%Y-%m-%dT%H:%M:%S%Z')
         return value
+
+
+SERVICE_NAME_ALIASES = {'runtime.sagemaker': 'sagemaker-runtime'}
+
+
+def handle_service_name_alias(service_name, **kwargs):
+    return SERVICE_NAME_ALIASES.get(service_name, service_name)
+
+
+SERVICE_RENAMES = {
```

Comment:
All of this should be clearly private, I don't think we want anything being built on this externally.